great-idea/services.html

```diff
@@ -16,92 +16,113 @@
-eleifend. Mauris euismod facilisis iaculis.
+<div class="middle-box"> 
+<h2>Web Development</h2>
 
-Learn More
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
+<button>Learn More</button>
 
+</div>
+</div>
+<div class="style3">
+<div class="middle-box"> 
 
-Android Development
+<h2>iOS Development</h2>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-Learn More
+<button>>Learn More</button>
+</div>
 
+<div class="middle-box"> 
+<h2>Android Development</h2>
 
-Some Facts About Our Services
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<button>Learn More</button>
+</div>
+</div>
+</section>
 
+<section class="last-part">
+  <div class="last-part-content">
+<h2>Some Facts About Our Services</h2>
 
-Awesome thing
-Amazing thing
-Cool thing
-Great thing
-
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<ul>
+   <li> Awesome thing </li>
+   <li> Amazing thing</li>
+    <li>Cool thing</li>
+   <li> Great thing</li>
+</ul>
+</div>
 <img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
-
+</section>
+<footer class="services-footer">
 Copyright Great Idea! 2018
-
+</footer>
+</div>
 </body>
 
 </html>
```

Comment:
The structure of your html is very good, it looks like you understand the concept of nesting of tags and their relationships.

Something interesting you can do which is very helpful when marking up your CSS is with your div container in your html you can add more classes to that div container. For example the index.html file which will be your home can have a class name home added to it like so 
div class=”container home”  and in your service page you can have div class=”container service”
And you can do a similar thing for all your pages.
This way every div that has a container class will share similar styling and can have different styling according to their different class names they have. Keeps your code more dry